parts/chapters/5.fodt

```diff
@@ -22072,17 +22050,17 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
+    <text:p text:style-name="P12281">Note that in some instances the numerical values entered need to be the actual values, as per the DIMENS keyword, or maximum values, for example the previously mentioned WELLDIMS keyword. If the requirement for the keyword is for the actual value and an incorrect value has been entered then this may or may not be directly trapped by the simulator, and may result in numerous error message associated with the data instead. Thus, if the error reporting messages in a given section are associated with data that is required to be declared via a RUNSPEC keyword, one should also verify that the RUNSPEC keyword has been correctly defined with the associated problematic data error messages. <text:s/>This is particularly true for the TABDIMS keyword in the RUNSPEC section where some of the variables are defined as the actual number of tables, whereas the length of the tables are a maximum value. To be clear, if NTSFUN on the TABDIMS keyword has been set to five, then there must be exactly five sets of relative permeability data entered in the PROPS section, otherwise data input errors will occur.</text:p>
     <text:section text:style-name="Sect1" text:name="Section5.2" text-protected="true">
  <text:section-source text:link-type="simple" xlink:href="sections/5/2.fodt" text:filter-name="OpenDocument Text Flat XML" text:section-name="Section2">
  </text:section-source>
 </text:section>
-<text:h text:style-name="P17544" text:outline-level="2"><text:bookmark-start text:name="__RefHeading___Toc46346_3701168388"/><text:span text:style-name="T918">K</text:span><text:span text:style-name="T1">eyword Definitions</text:span><text:bookmark-end text:name="__RefHeading___Toc46346_3701168388"/></text:h>
+<text:h text:style-name="P17544" text:outline-level="2"><text:bookmark-start text:name="__RefHeading___Toc46346_3701168388"/>K<text:span text:style-name="T1">eyword Definitions</text:span><text:bookmark-end text:name="__RefHeading___Toc46346_3701168388"/></text:h>
```

Comment:
@gdfldm Good observation. It seems like the the type "T1" of the second span is not revision id span. It is defined at line 16747 in the original file:
```
  <style:style style:name="T1" style:family="text">
   <style:text-properties fo:language="en" fo:country="US"/>
  </style:style>
```
observe that the style does not contain an attribute `officeooo:rsid`. The script only removes span tags that have styles with revsions ids. It would be interesting to know to the purpose of the "T1" style..